Formula/openjdk@8.rb

```diff
@@ -49,15 +49,16 @@ class OpenjdkAT8 < Formula
     depends_on "libxtst"
   end
 
-  # Oracle doesn't serve JDK 7 downloads anymore, so use Zulu JDK 7 for bootstrapping.
+  # Oracle doesn't serve JDK 8 downloads anymore, so we use Zulu JDK 8 for bootstrapping.
```

Comment:
Are we sure we want to switch to JDK 8 for bootstrapping? AFAIR it's recommended to use the previous version for this.